server/@types/models/Referral.ts

```diff
@@ -1,8 +1,11 @@
+import type { CourseOffering } from './CourseOffering'
+import type { Person } from './Person'
+
 type Referral = {
   id: string // eslint-disable-next-line @typescript-eslint/member-ordering
-  offeringId: string
-  prisonNumber: string
-  referrerId: string
+  offeringId: CourseOffering['id']
+  prisonNumber: Person['prisonNumber']
+  referrerId: Express.User['userId'] // no idea which of the three User/UserDetails interfaces is best to use here
```

Comment:
I think this one is probably fine - I was going to suggest using the `User` model but that doesn't have the `token` on it yet. This one'll do us